page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/PageManager.java

```diff
@@ -29,13 +29,16 @@
 public class PageManager {
 
     private static final Logger LOG = LoggerFactory.getLogger(PageManager.class);
-    private static final Map<Class<? extends Page>, Map<Field, String>> PAGES_REPOSITORY = new HashMap<>();
+
+    private static final ThreadLocal<Map<Class<? extends Page>, Map<Field, String>>> PAGES_REPOSITORY
+            = ThreadLocal.withInitial(() -> new HashMap<Class<? extends Page>, Map<Field, String>>());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. ([squid:S1612](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1612))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOEucrQ7oSxgxngA&open=AWeIHOEucrQ7oSxgxngA)

[ISSUE_KEY]: AWeIHOEucrQ7oSxgxngA